src/clib/CMakeLists.txt

```diff
@@ -219,7 +219,7 @@ endif ()
 # Hack to circumvent IBM XL (16.1.1-3) internal compiler error by
 # disabling optimization for pioc_support.c
 cmake_host_system_information (RESULT FQDN_SITENAME QUERY FQDN)
-if (FQDN_SITENAME MATCHES "^.*summit[.]olcf")
+if (FQDN_SITENAME MATCHES "^.*[.]olcf")
```

Comment:
Do you think we should add these flags for just ascent and summit or all olcf machines (for XL)?

Reply:
I think it should be all: (1) it wasn't easy to find this (the flags do not show up in config_compilers.xml and Depends.* files and GH search does not recursively search in submodules), and (2) any XL version from 16.1.1-1 to 16.1.1-10 errors out with ICE.

Reply:
ok, sounds good.